classes/simplemap.php

```diff
@@ -732,25 +732,25 @@ function google_map_js_script() {
 			var units                    = '<?php echo esc_js( $options['units'] ); ?>';
 			var limit                    = '<?php echo esc_js( $options['results_limit'] ); ?>';
 			var plugin_url                = '<?php echo esc_js( SIMPLEMAP_URL ); ?>';
-			var visit_website_text        = '<?php echo apply_filters( 'sm-visit-website-text', __( 'Visit Website', 'simplemap' ) ); ?>';
-			var get_directions_text        = '<?php echo apply_filters( 'sm-get-directions-text', __( 'Get Directions', 'simplemap' ) ); ?>';
-			var location_tab_text        = '<?php echo apply_filters( 'sm-location-text', __( 'Location', 'simplemap' ) ); ?>';
-			var description_tab_text    = '<?php echo apply_filters( 'sm-description-text', __( 'Description', 'simplemap' ) ); ?>';
-			var phone_text                = '<?php echo apply_filters( 'sm-phone-text', __( 'Phone', 'simplemap' ) ); ?>';
-			var fax_text                = '<?php echo apply_filters( 'sm-fax-text', __( 'Fax', 'simplemap' ) ); ?>';
-			var email_text                = '<?php echo apply_filters( 'sm-email-text', __( 'Email', 'simplemap' ) ); ?>';
+			var visit_website_text        = '<?php echo apply_filters( 'sm-visit-website-text', __( 'Visit Website', 'SimpleMap' ) ); ?>';
+			var get_directions_text        = '<?php echo apply_filters( 'sm-get-directions-text', __( 'Get Directions', 'SimpleMap' ) ); ?>';
+			var location_tab_text        = '<?php echo apply_filters( 'sm-location-text', __( 'Location', 'SimpleMap' ) ); ?>';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 127 characters

<!-- Comment by [SideCI](https://sideci.com) -->